app/components/user-form/component.js

```diff
@@ -0,0 +1,43 @@
+import Ember from 'ember';
+import { userValidations } from 'bsc-ember/validations/user';
+import Changeset from 'ember-changeset';
+import lookupValidator from 'ember-changeset-validations';
+
+const { Component, inject: { service } } = Ember;
+
+export default Component.extend({
+  paperToaster: service(),
+  currentUser: service(),
+  router: service(),
+  store: service(),
+  genders: ['male', 'female'],
```

Comment:
Translation is missing

Reply:
It is not: https://github.com/MattSkrobis/bsc-ember/pull/31/files/a5fe7e3b7198b88083cae2e65972f7d2feb45f3c#diff-a448ae814c0d31ba194bde0eb91f8b6bR13